### PR TITLE
Exercise 10 ~ GradeSchool

### PR DESCRIPTION
**Pull request by inammathe**
Exercise 10 ~ GradeSchool #59

### CODE REVIEW

**Comment by masters3d** [comment]
-> An implementation for 'grade-school' was found, but config.json does not reference this exercise.
-> The implementation for 'grade-school' is missing a README.

**Comment by inammathe** [comment]
from https://github.com/exercism/powershell/issues/59

>  _Do not touch_, as it will not be merged:
> .\config.json
> .\exercises\grade-school\README.md (autogenerated)

**Comment by masters3d** [comment]
that must be old. this pr has a config.

**Comment by masters3d** [comment]
https://github.com/exercism/powershell/commit/94a4c08daa8fa9622ae5a8ffe700a9d1657d07a5#diff-06b2d3b23dce96e1619d2b53d6c947ec

**Comment by masters3d** [comment]
@kytrinyx who can add mantainers? I can help a bit here

**Comment by inammathe** [comment]
Thanks @masters3d 

Are we happy with using classes? I wasn't really sure as it would be the first exercise to use this fomat i.e. expecting the user to write two PowerShell classes to complete the solution. 

Not to mention classes were only introduced in 5.0. I am unsure if we are supposed to be supporting old PS versions.

**Comment by masters3d** [comment]
PowerShell 5 is reasonable but at some point it be good if we moved the whole track to 6

**Comment by gyssels** [comment]
> that must be old. this pr has a config.

Not old, I have a code that takes the text from "problem-specifications" for the ".md" file and makes sure that nothing is lost in copy& paste... and a config.json autogenerated with the submission of new exercise folder(attention given that a Guid should never be changed after a exercism is published, and the write order and exercism dependencies).

I always run those script when someone commit new exercices.

**Comment by masters3d** [comment]
That should be part of the PR or a github action then? Travis fails if they are not present. I really like the idea, it could also be used for other tracks, that is the common issue for new contributors.

**Comment by inammathe** [comment]
So revert https://github.com/exercism/powershell/pull/100/commits/64cb5195a0ad8a453f73e33ee92f144e39f62cf1? I made that config change by hand (generated the guid with `New-Guid`).

**Comment by masters3d** [comment]
@gyssels what are the next steps?

**Comment by kytrinyx** [comment]
> who can add mantainers?

That's me, at the moment. We have a bot that sometimes works when you add yourself to the `config/maintainers.json` file, but if it doesn't within a minute or so, then the next step would be to ping me. You're on the powershell team now :)

File: config.json [hunk 1]
```diff
@@ -102,6 +102,17 @@
         "filtering",
         "strings"
       ]
+    },
+    {
+      "slug": "grade-school",
+      "uuid": "2daa9e3e-65c1-4fc3-b6ee-6f477a198fec",
```
Comment (inammathe) [inline config.json:108]: Done by hand (`new-guid`).

**Comment by masters3d** [comment]
Please remove the WIP if ready. @gyssels are you okay If I merge this?

**Comment by masters3d** [comment]
I am going to merge this. If there is a issue we can revert it. Thank you  @inammathe ! Sorry about the delay.

**Pull request merged**